sound/hybrid_font.h

```diff
@@ -320,9 +320,15 @@ class HybridFont : public SaberBase, public Looper {
               float s = (rss - font_config.ProffieOSMinSwingAcceleration) / font_config.ProffieOSMaxSwingAcceleration;
 	      effect->SelectFloat(s);
```

Comment:
well wouldn't that mean it would need to be repeated for both EFFECT_ACCENT_SWING and EFFECT_ACCENT SLASH cases? Is it not neater to keep it in just one occurrence where it is?
Or again, is this lending itself to all of these changes somehow in a new class or function?
This is starting to grow out of my ability to brain this.